scripts.js

```diff
@@ -0,0 +1,38 @@
+(function () {
```

Comment:
This makes it a self executing block of code, and prevents anything inside of it from being in the global namespace.  Access to the variables and functions inside the closure must be made public through some sort of namespace interaction, or else nothing can 'talk' to the code inside the block.  It's pretty much a scoping issue, which causes similar behavior to Java's Public / Private methods inside of a larger class.

Use of this form is normal if you want to prevent your code from polluting the global space, or if you needed it to self execute on parse, rather than calling the function via some other block of code.